src/App.tsx

```diff
@@ -82,7 +82,7 @@ const PackageEditorPage = lazyImport(async () => {
 const ReleasesPage = lazyImport(() => import("@/pages/releases"))
 const ReleaseDetailPage = lazyImport(() => import("@/pages/release-detail"))
 const ReleaseBuildsPage = lazyImport(() => import("@/pages/release-builds"))
-const PreviewBuildPage = lazyImport(() => import("@/pages/preview-build"))
+const ReleasePreviewPage = lazyImport(() => import("@/pages/preview-release"))
```

Comment:
Follow the convention of variable name matching with the page name, in the same order